bedrock/firefox/templates/firefox/nothing-personal/index.html

```diff
@@ -149,89 +227,55 @@ <h4>You’re great — probably.</h4>
-      <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
-      <br>
-      <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
-      <br>
-      <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
-
-      {{ apple_app_store_button(href=ios_url, class_name='app-store-badge') }}
-      {{ google_play_button(href=android_url, class_name='play-store-badge') }}
-
-      <p>Ok, bye.</p>
-    </div>
-
-    <div class="c-sign-off">
-      <div class="mzp-l-content c-nothing-personal">
-        <p>Nothing personal. Just browsing.</p>
-        <div class="c-sign-off-wordmark">
-          <img loading="lazy" src="{{ static('img/firefox/nothing-personal/firefox-by-mozilla-wordmark-white.svg') }}" alt="" width="180" height="40" class="c-header-logo">
         </div>
       </div>
-    </div>
     </section>
-    <aside>
-      <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" aria-label="Link to a Youtube video" target="_blank" rel="external noopener" data-link-text="Folder"><img src="{{ static('img/firefox/nothing-personal/folder-icon.svg') }}" alt="">Folder <br> <.< </a>
-      <a href="https://open.spotify.com/playlist/0vvXsWCC9xrXsKd4FyS8kM" aria-label="Link to a Spotify playlist" target="_blank" rel="external noopener" data-link-text="Focus playlist"><img src="{{ static('img/firefox/nothing-personal/music-icon.svg') }}" alt="">focus <br> playlist</a>
-      <div class="c-trash" aria-hidden="true">
-        <img src="{{ static('img/firefox/nothing-personal/bin-icon.svg') }}" alt="">
-        <span>MyLife</span>
-      </div>
-    </aside>
-
+    <!-- final CTAs -->
   </main>
+  <aside class="c-aside">
+    <a class="c-aside-link" href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" aria-label="Link to a Youtube video" target="_blank" rel="external noopener" data-link-text="Folder"><img src="{{ static('img/firefox/nothing-personal/folder-icon.svg') }}" alt="">Folder <br> <.< </a>
+    <a class="c-aside-link" href="https://open.spotify.com/playlist/0vvXsWCC9xrXsKd4FyS8kM" aria-label="Link to a Spotify playlist" target="_blank" rel="external noopener" data-link-text="Focus playlist"><img src="{{ static('img/firefox/nothing-personal/music-icon.svg') }}" alt="">focus <br> playlist</a>
+    <a class="c-aside-link" href="https://www.youtube.com/watch?si=RHt_agrKQEODTPRP&v=F-nFQryDB0s&feature=youtu.be" aria-label="Introducing the new Firefox...jk. It's red pandas" target="_blank" rel="external noopener" data-link-text="MyLife">
```

Comment:
Nit: can remove some of the additional tracking params from this URL:
```suggestion
    <a class="c-aside-link" href="https://www.youtube.com/watch?v=F-nFQryDB0s" aria-label="Introducing the new Firefox...jk. It's red pandas" target="_blank" rel="external noopener" data-link-text="MyLife">
```